wordpress-mobile/WordPress-Android
# Api26/jobschedule stats service

Following the same pattern and structure started in #7603, this PR splits `StatsService` in its two versions, a `Service` for API below level 26, and `JobService` for API 26 and beyond.

To test:
1. Start the app
2. Go to Stats
3. Check all the sections are loaded correctly
4. Test pull down to refresh works
5. Try different views by choosing Insights, Day, Month, etc. from the drop down filter.

Use please both platform 26+ and below, and confirm that you see the same data for your sites on both platforms.

cc @nbradbury 
also cc @daniloercoli would you give it a try? 🙇
